src/lib/dom.iterable.d.ts

```diff
@@ -9,5 +9,9 @@ interface NodeList {
 }
 
 interface NodeListOf<TNode extends Node> {
+    keys(): IterableIterator<number>;

```

Comment:
Why only `keys()` ? It makes more sens to add them all. I'm not sure I'm reading your message correctly.